qctrlopencontrols/dynamic_decoupling_sequences/dynamic_decoupling_sequence.py

```diff
@@ -70,6 +69,23 @@ class DynamicDecouplingSequence:
     ------
     qctrlopencontrols.exceptions.ArgumentsValueError
         is raised if one of the inputs is invalid.
+
+    Notes
+    -----
+    Dynamical decoupling sequence (DDS) is canonically defined as a series of
+    :math:`n`-instantaneous unitary operations, often :math:`\pi`-pulses, executed
+    at time offsets :math:`\{t_j\}_{j=1}^n` over the time interval with a total
+    duration :math:`\tau`. The :math:`j`-th operation applied at time
+    :math:`t_j` can be parameterized as
+
+    .. math::
+        U_j = \exp\left[-\frac{i}{2}(\omega_j \cos \phi_j \sigma_x + \omega_j\sin \phi_j\sigma_y
+        + \delta_j\sigma_z)\right] \;,
+
+    Note that in practice all DDSs typically have a :math:`X_{\pi/2}` operation at the start
+    :math:`t_0` and end :math:`t_\tau` of the sequence. This is because it is assumed that the
```

Comment:
Hrm, sorry to be pedantic about such a small thing, but I think using t=0 and t=\tau would be more clear (and defining \tau in the `duration` description).

Reply:
ah, right. That makes sense. Will fix.